docs/mobile-apps/automated-testing/espresso-xcuitest/real-devices.md

```diff
@@ -24,51 +17,76 @@ Our Sauce Runner for Real Devices test runner enables you to integrate Espresso
+Starting test run on US data center...
+```
+7. View your results on Sauce Labs. Go to **Automated** > **Test Results** > select the **Real Devices** radio button.
 
-#### Configuring XCUITest Files for Sauce Runner
+<br/>
+
+:::tip Help Command
+For a full list of test specifications, execute the `help` command:
 
-Once you've written your iOS app test in ObjectiveC/Swift, you'll need to build it as an .ipa file for use with Sauce Runner for Real Devices. For instructions, see **Creating an .ipa File** below.
+```sh
+java -jar saucelabs-native-test-runner.jar help
+```
+:::
 
-Under **iOS Deployment Target**, ensure that you set the same iOS version for both the app and your test runner. If these don't match, your tests will run locally, but fail when you run them against Sauce Labs real devices. From your Xcode **Build Settings**:
+## Setting Up Your Environment
 
-1. **Select Your App Project**: Select the Project you want to build, and under **Build Settings**, set the iOS Deployment Target to the iOS operating system version you want to use in your test. All target outputs of this project, including the application and your test runner, will be set to the same operating system version.
+### Espresso (Android)
 
-2. **Select Your App Target**: Select the **Target** for your Project, and under **Build Settings**, set the **iOS Deployment Target** to the iOS operating system version you want to use in your test. This will also overwrite the Build Settings at the Project level to that operating system version, so if you use this method, be aware that your Targets can become out of sync with each other and the Project settings, and your tests will break. If you change the iOS version for one target output, you may want to build the Project again to make sure all your targets are in sync.
+There are two ways you can run Espresso and UI Automator tests against Sauce Labs real devices:
+
+* By using our test runner, which is parameterized for use in CI/CD environment
+* By using our [web interface](https://app.saucelabs.com)
+
+### XCUITest (iOS)
 
+Below are instructions on how to create an .ipa file using self-signed application files and .app files with our XCUITest test runner.
 
-#### Creating an .ipa File
+All of the build types listed below are supported, since we use a resigning process to embed our own provisioning profile into the uploaded .ipa file. With your own profile in place, the app will be allowed to install and launch on the cloud devices. The functionality of the app itself remains completely unmodified.
 
-Learn how to create an .ipa file using self-signed application files and .app files with our XCUITest test runner.
+You can also use .ipa files generated by build tools outside of Xcode.
 
 :::tip
 If you need to maintain your own provisioning profile in the .ipa file, you should disable the **App Resigning** option when you set the Device Settings for your app in the cloud.
```

Comment:
Thank you for letting me know! This section def needs some love. 

Since it contains a mix of Appium and XCUITest info, I've pulled it out into its own page temporarily, then sort out which is which ... https://docs.saucelabs.com/mobile-apps/automated-testing/ipa-files